ref/net46/Microsoft.Build.Framework/Microsoft.Build.Framework.cs

```diff
@@ -571,12 +580,20 @@ public partial struct EvaluationLocation
         public Microsoft.Build.Framework.Profiler.EvaluationLocation WithFileLineAndCondition(string file, System.Nullable<int> line, string condition) { throw null; }
         public Microsoft.Build.Framework.Profiler.EvaluationLocation WithFileLineAndElement(string file, System.Nullable<int> line, Microsoft.Build.Framework.IProjectElement element) { throw null; }
         public Microsoft.Build.Framework.Profiler.EvaluationLocation WithGlob(string globDescription) { throw null; }
+        public Microsoft.Build.Framework.Profiler.EvaluationLocation WithParentId(System.Nullable<int> parentId) { throw null; }
+    }
+    public partial class EvaluationLocationIdAgnosticComparer : System.Collections.Generic.IEqualityComparer<Microsoft.Build.Framework.Profiler.EvaluationLocation>
```

Comment:
Make private.

Reply:
Moved to where the profiler logger is and made internal

---
In reply to: [157260451](https://github.com/Microsoft/msbuild/pull/2813#discussion_r157260451) [](ancestors = 157260451)